test/functional/Java8andUp/src/org/openj9/test/java/lang/Test_StringBuilder.java

```diff
@@ -265,6 +265,49 @@ public void test_codePointCount() {
 		AssertJUnit.assertTrue("wrong count 2", sb.codePointCount(1, 4) == 2);
 	}
 
+	/**
+	 * @tests java.lang.StringBuilder#offsetByCodePoints(int, int)
+	 */
+	@Test
+	public void test_offsetByCodePoints() {
+		StringBuilder s1 = new StringBuilder("A\ud800\udc00C");
+		AssertJUnit.assertTrue("wrong offset 0, 0", s1.offsetByCodePoints(0, 0) == 0);
```

Comment:
These would be better written using `assertEquals()` which provides more information when it fails.